src/frontend/src/pages/DesignReviewSummaryModal.tsx

```diff
@@ -187,20 +39,22 @@ const DRCSummaryModal: React.FC<DRCSummaryModalProps> = ({ open, onHide, designR
         <Grid item>
           <Grid container direction="row" alignItems="center" justifyContent="center" columnSpacing={1}>
             <Grid item xs={3}>
-              <Pill icon={<AccessTimeIcon />} text={designReview.meetingTimes.map(meetingTimePipe).join(', ')} />
+              <Pill icon={<AccessTimeIcon />} isLink={false} displayText={meetingTimePipe(designReview.meetingTimes[0])} />
             </Grid>
             <Grid item xs={3}>
-              <Pill icon={<LocationOnIcon />} text={designReview.location ?? 'Online'} />
+              <Pill icon={<LocationOnIcon />} isLink={false} displayText={designReview.location ?? 'Online'} />
             </Grid>
             <Grid item xs={3}>
-              <LinkPill
+              <Pill
+                isLink
                 icon={<DescriptionIcon />}
                 linkText={designReview.docTemplateLink ?? ''}
                 displayText={designReview.docTemplateLink ? 'docs' : 'No Doc'}
               />
             </Grid>
             <Grid item xs={3}>
-              <LinkPill
+              <Pill
+                isLink
                 icon={<VideocamIcon />}
                 linkText={designReview.zoomLink ?? ''}
                 displayText={designReview.zoomLink ? 'zoom.us' : 'No Zoom'}
```

Comment:
We should probably make the link texts like `Zoom Link` and `Questions Doc` to avoid confusion with the link text being the actual url